packages/docs/src/routes/community/menu.md

```diff
@@ -5,5 +5,5 @@
 - [Groups](groups/index.mdx)
 - [Projects](projects/index.mdx)
 - [Values](values/index.mdx)
-- [Media](../media/index.tsx)
-- [Showcase](../showcase/index.tsx)
+- [Media](../media/index.mdx)
+- [Showcase](../showcase/index.mdx)
```

Comment:
This resolve the 404 error but we can't access to the file. 😅

@zanettin could provide you more feedback.

Thanks for the PR!

Reply:
Hi @cunzaizhuyi 👋  thx for this PR 🙏 
It works but i leave the decision on this to @manucorporat since the core team has plans to refactore some part of the doc 🎉 

@kemilbeltre : thanks for the ping 🙏  not sure what you mean with `access to the file` 👼

Reply:
`access the file`: I meant that we cannot go to that file by clicking on the link in the `menu.md` file, because the extension is not `.mdx`.

Reply:
ah gotcha 👍  yes that is true. But afaik there is no other way to link to these files except of `../../media` 🙈